crates/contracts/artifacts/Placeholder.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
For other files in this folder we minimized the json. Not super important, just wanted to mention.

Reply:
Nice catch! We should use `jq -c` for compact output. I had it at somepoint but removed it to debug the final output. Will put it back in.